contracts/JetStaking/JetStakingV3.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import "./JetStakingV2.sol";
+
+/**
+ * @title JetStakingV3
+ * @author Aurora Team
+ *
+ * @dev Implementation of Jet staking contract
+ *
+ *      This contract implements the staking mechanics for AURORA ERC20 token.
+ *      A user can stake any amount of AURORA tokens, and get rewarded in both
+ *      AURORA and other stream tokens based on the rewards schedules.
+ *      Stream rewards can be claimed any time however AURORA can't be claimed
+ *      unless the user unstakes his full/partial amount of shares.
+ *
+ *      This contract is AdminControlled which has a tremendous power. However
+ *      hopfully it be governed by a community wallet.
+ */
+contract JetStakingV3 is JetStakingV2 {
+    // RPS_MULTIPLIER = Aurora_max_supply x weight(1000) * 10 (large enough to always release rewards) =
+    // 10**9 * 10**18 * 10**3 * 10= 10**31
+    uint256 private constant RPS_MULTIPLIER = 1e31;
```

Comment:
Can we change the visibility of this constant, so it would not be required to define it again for future upgrades?